torch/legacy/nn/MarginRankingCriterion.py

```diff
@@ -60,7 +60,7 @@ def updateGradInput(self, input, y):
             self.mask = self.mask.resize_as_(input[0]).copy_(dist)
             mask = self.mask
 
-            torch.ge(dist, 0, out=mask)
+            torch.ge_t(dist, 0, out=mask)
```

Comment:
Overloading the behavior of a function on the type of value being produced is generally suspicious to me because an output parameter is secretly providing an input to the function -- what type should be returned. A argument to control the return type might be ok, but is going require its own changes to cwrap/TensorLib since it is a new case. I was under the impression that the release this would go in would introduce other breaking changes as well. Is this even used?